src/swell/configuration/jedi/interfaces/geos_marine/observations/adt_swot_nadir.yaml

```
@@ -0,0 +1,70 @@
+obs space:
+  name: adt_swot_nadir
+  obsdatain:
+    engine:
+      type: H5File
+      obsfile: '{{cycle_dir}}/adt_swot_nadir.{{window_begin}}.nc4'
+  obsdataout:
+    engine:
+      type: H5File
+      obsfile: '{{cycle_dir}}/{{experiment_id}}.adt_swot_nadir.{{window_begin}}.nc4'
+  simulated variables: [absoluteDynamicTopography]
+obs operator:
+  name: ADT
+obs error:
+  covariance model: diagonal
+obs filters:
+- filter: Domain Check
+  where:
+  - variable: {name: GeoVaLs/sea_area_fraction}
+    minvalue: 0.9
+- filter: Domain Check
+  where:
+  - variable: { name: GeoVaLs/sea_surface_temperature}
+    minvalue: 15.0
+- filter: Background Check
+  absolute threshold: 0.2
+- filter: Domain Check
+  where:
+  - variable: {name: GeoVaLs/sea_floor_depth_below_sea_surface}
+    minvalue: 500
+# - filter: Perform Action
+#   action:
+#     name: assign error
+#     error function:
+#       name: ObsFunction/LinearCombination
+#       options:
+#         variables: [{name: GeoVaLs/mesoscale_representation_error},
+#                     {name: ObsError/absolute_dynamic_topography}]
+#         # variables: [mesoscale_representation_error@GeoVaLs,
+#         #             absolute_dynamic_topography@ObsError]
+#         coefs: [0.1,
+#                 0.5]
+# - filter: BlackList
+#   where:
+#   - variable:
+#       name: MetaData/latitude
+#     minvalue: -65
+#     maxvalue: -30
+#   - variable:
+#       name: MetaData/longitude
+#     minvalue: -125
+#     maxvalue: -90
+# - filter: BlackList
+#   where:
+#   - variable:
+#       name: MetaData/latitude
+#     minvalue: -65
+#     maxvalue: -30
+#   - variable:
+#       name: MetaData/longitude
+#     minvalue: 60
+#     maxvalue: 110
+# - filter: Domain Check
+#   where:
+#   - variable: { name: GeoVaLs/sea_ice_area_fraction}
+#     maxvalue: 0.00001
+- filter: Domain Check
+  where:
+  - variable: { name: GeoVaLs/distance_from_coast}
+    minvalue: 100e3
```

```suggestion
- filter: Domain Check
  where:
  - variable: { name: GeoVaLs/distance_from_coast}
    minvalue: 100e3
{% if 'cice6' in marine_models %}
- filter: Domain Check
  where:
  - variable: { name: GeoVaLs/sea_ice_area_fraction}
    maxvalue: 0.00001
{% endif %}
```

Since sea ice fraction is CICE6 state dependent that should only be active when `cice6` is used. Jinja2 formatting allows us to do this.